pubsub/observer.go

```diff
@@ -0,0 +1,26 @@
+package pubsub
```

Comment:
@nivBlox please add an issue to refactor this pubs struct to chan

Reply:
https://github.com/ethereum/eth2-ssv/issues/88